spec/models/quote_type_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe QuoteType, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_length_of(:name).is_at_most(255) }
+
+  it { should validate_presence_of(:order) }
+  it { should validate_length_of(:order).is_at_most(255) }
+  # should order be number in string
```

Comment:
what is this command about?

Reply:
It validates the maximum field length, 255.

Reply:
`  # should order be number in string` this one, this comment is not necessary